moveit_ros/planning/planning_scene_monitor/CMakeLists.txt

```
@@ -1,18 +1,29 @@
 set(MOVEIT_LIB_NAME moveit_planning_scene_monitor)
 
 add_library(${MOVEIT_LIB_NAME}
-  src/planning_scene_monitor.cpp
+  # src/planning_scene_monitor.cpp
   src/current_state_monitor.cpp
-  src/trajectory_monitor.cpp)
+  # src/trajectory_monitor.cpp
+)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 target_link_libraries(${MOVEIT_LIB_NAME}
   moveit_robot_model_loader
   moveit_collision_plugin_loader
-  ${catkin_LIBRARIES}
+  ${rclcpp_LIBRARIES}
   ${Boost_LIBRARIES})
 
-add_executable(demo_scene demos/demo_scene.cpp)
-target_link_libraries(demo_scene ${MOVEIT_LIB_NAME} ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+ament_target_dependencies(${MOVEIT_LIB_NAME}
+  moveit_robot_model_loader
+  moveit_collision_plugin_loader
+  ${rclcpp_LIBRARIES})
+
+#TODO: (anasarrak) Fix this once moveit_ros_planning is finished
```

```suggestion
#TODO: (anasarrak) Fix this once moveit_ros_planning is converted to ROS2
```

If you forget to do this, let's have descriptive comments so others know what you meant by 'finished'